package.json

```diff
@@ -6,8 +6,8 @@
     "postinstall": "scripts/copy-node-modules.sh",
     "build": "echo 'WARNING: `npm run build` in edx-platform is experimental. Use at your own risk.' && npm run webpack && npm run compile-sass",
     "build-dev": "echo 'WARNING: `npm run build-dev` in edx-platform is experimental. Use at your own risk.' && npm run webpack-dev && npm run compile-sass-dev",
-    "webpack": "NODE_ENV=${NODE_ENV:-production} \"$(npm bin)/webpack\" --config=${WEBPACK_CONFIG_PATH:-webpack.prod.config.js}",
-    "webpack-dev": "NODE_ENV=development \"$(npm bin)/webpack\" --config=webpack.dev.config.js",
```

Comment:
Could you use `$(npm root)/.bin` here too?

Reply:
This is not necessary as npm [adds it to the path](https://docs.npmjs.com/cli/v8/commands/npm-run-script?v=true#description):

>In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix.

Reply:
👍🏻